Source/Core/Common/Arm64Emitter.cpp

```diff
@@ -3893,6 +3893,8 @@ void ARM64FloatEmitter::ABI_PushRegisters(BitSet32 registers, ARM64Reg tmp)
 
   if (bundled_loadstore && tmp != ARM64Reg::INVALID_REG)
   {
+    DEBUG_ASSERT_MSG(DYNA_REC, Is64Bit(tmp), "Expected an XReg temporary!");
```

Comment:
The term `XReg` isn't familiar to me. How about:

```suggestion
    DEBUG_ASSERT_MSG(DYNA_REC, Is64Bit(tmp), "Expected a 64-bit temporary register!");
```

There's also the question of whether it would be useful to have this in release builds... But if you feel having it in debug builds is sufficient, I don't mind.

Reply:
It could go either way, but I thought the rest of the code exclusively using 64-bit registers should be self-documenting enough to make this assertion not necessary for release builds.